packages/mui-base/src/SnackbarUnstyled/SnackbarUnstyled.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { createRenderer, createMount, describeConformanceUnstyled } from 'test/utils';
+import SnackbarUnstyled, { snackbarUnstyledClasses as classes } from '@mui/base/SnackbarUnstyled';
+
+describe('<SnackbarUnstyled />', () => {
+  const { render } = createRenderer();
+  const mount = createMount();
+
+  describeConformanceUnstyled(
+    <SnackbarUnstyled open>
+      <div />
+    </SnackbarUnstyled>,
+    () => ({
+      classes,
+      inheritComponent: 'div',
+      render,
+      mount,
+      refInstanceof: window.HTMLDivElement,
+      muiName: 'BaseSnackbar',
+      slots: {
+        root: {
+          expectedClassName: classes.root,
+        },
+      },
+      skip: ['componentsPropsCallbacks', 'ownerStatePropagation'], // not implemented yet
```

Comment:
```suggestion
```

We don't want to skip tests for new components, we can implement the features right away :)

Reply:
Done.